pathplannerlib/src/main/java/com/pathplanner/lib/path/PathPlannerPath.java

```diff
@@ -38,6 +39,7 @@ public class PathPlannerPath {
    * @param globalConstraints The global constraints of the path
    * @param goalEndState The goal end state of the path
    * @param reversed Should the robot follow the path reversed (differential drive only)
+   * @param previewStartingRotation The settings used for previews in the UI
    */
   public PathPlannerPath(
```

Comment:
Can you add an overload for this constructor that doesn't need a previewStartingRotation? This constructor can be used for on-the-fly generation as well so it would be nice to not need to give it one.